client_app/src/components/shelter/UpcomingRequests.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "../../styles/UpcomingRequests.css";
+
+const UpcomingRequests = ({ shifts }) => {
+  return (
+    <div className="upcoming-requests">
```

Comment:
Currently, the page is not rendered, so I could not see the design of this page.